src/pages/Products/Client.tsx

```
@@ -0,0 +1,152 @@
+                    </div>
+                </div>
+                <div class="call-to-action slideUp">
+                    <div class="text">
+                        <h3>{t("boltz_client_cta_title")}</h3>
+                        <p>{t("boltz_client_cta_subtitle")}</p>
+                    </div>
+                    <div class="buttons-line">
+                        <a
+                            class="btn-primary"
+                            href="https://client.docs.boltz.exchange/"
+                            target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)